pkg/local_object_storage/metabase/select.go

```diff
@@ -121,7 +121,7 @@ func (db *DB) selectObjects(tx *bbolt.Tx, cnr cid.ID, fs object.SearchFilters, c
 		addr.SetContainer(cnr)
 		addr.SetObject(id)
 
-		if objectStatus(tx, addr, currEpoch) > 0 {
+		if objectStatus(tx, addr, currEpoch) > 0 && !objectLocked(tx, cnr, id) {
```

Comment:
did you think about return specific status from `objectStatus` for locked objects?

Reply:
what is your idea exactly? in search operations be ready to return objects _and_ status code if _any_ locked objects have been found?

Reply:
oh, sorry, got your wrong

Reply:
found your idea usefull, check, please